dev-client/src/screens/ManageTeamMemberScreen.tsx

```diff
@@ -81,53 +86,69 @@ export const ManageTeamMemberScreen = ({
     navigation.pop();
   }, [dispatch, projectId, userId, selectedRole, navigation]);
 
+  const handleMissingProject = useHandleMissingSiteOrProject();
+  const handleMissingUser = useCallback(() => {
+    navigation.pop();
+    if (isFlagEnabled('FF_offline')) {
+      syncNotifications.showError();
+    }
+  }, [navigation, syncNotifications]);
+  const requirements = [
```

Comment:
I think we should find a way to avoid recomputing a new value every render cycle for requirements, since the consequences for re-rendering the Restrict component + potentially re-evaluating its child function seem unclear to me.

Reply:
Thinking more about it, I actually don't think this is really an issue currently. If the Screen is re-rendering, the `ScreenDataRequirements` component will also re-render (unless we memoize `ScreenDataRequirements` component, but I don't think that's necessary (and side note: empirically it looks like if a memoized component has children it'll still re-render when the parent does, maybe unless its children are also memoized? Not sure)) 

But the solution we talked through works and I don't think it's any harm, just adds complexity is probably not needed currently.

Reply:
Fine by me -- entirely possible I was being over-vigilant here. I'll let you make the call on whether to keep this logic before merging.